### PR TITLE
:sparkles: Contact page form

### LINKED ISSUE #184: 🎨 Create Contact Page Form
#  Create the contact page form section 
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a nice user-friendly contact form on my Contact page to facilitate easy communication with my users.

Design:

contact form section

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```
 
### Technical Analysis

This feature can be found in `libs\features\pages\contact-page` inside a components folder. The component's name is contact-page-form-section. Use [Reactive Forms](https://angular.io/guide/reactive-forms) to process user input i.e. add logic to submit the form/collect the user data (don't send it anywhere). 

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
5. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by nicholasndolo**
# Description

A contact form on the Contact page that facilitates easy communication with my users.
I have collaborated with Damaris Nduku and Chrispine Odour.

Fixes # (issue) -: Fixes #184 
## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Screen
Tablet
Phone

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

### CODE REVIEW

File: apps/elewa-website/src/app/app.routing.ts [hunk 1]
```diff
@@ -46,6 +46,14 @@ export const ELEWA_WEBSITE_ROUTES: Route[] = [
       ),
   },
 
+  {
+    path: 'contact-form',
+    loadChildren: () =>
+      import('@elewa-website/features/pages/contact-page').then(
+        (m) => m.ContactPageModule
+      ),
+  },
```
Comment (LemmyMwaura) [inline apps/elewa-website/src/app/app.routing.ts:55]: Should not be a route

**Review by LemmyMwaura — COMMENTED** [review]
Very clean work 👍, Unfortunately, I can't merge this as it's already been resolved.

**Comment by nicholasndolo** [comment]
No problem, thanks!

**Comment by LemmyMwaura** [comment]
Attempt #10 @nicholasndolo

**Pull request closed without merging**